changelog/v1.5.0-beta13/glooctl-demo-federation.yaml

```
@@ -0,0 +1,6 @@
+changelog:
+  - type: NEW_FEATURE
+    description: |
```

nit, unless we have a good reason, we should prefer:
```suggestion
    description: >-
```